GUI/Components/Panels/AdvancedSearch.py

```diff
@@ -83,7 +86,6 @@ def addEmployee(self):
 
     def searchAdvanced(self, *args):
         employees = DB.search(**self.getAllNonEmptyEntries())
-        self.master=typing.cast('Search',self.master)
```

Comment:
Why is this removed?

Reply:
Because the cast was already done in the init function, and self.master is never changed after that. No need to double cast it

Reply:
do you have typechecking turned on in vscode? for me, it still complained without the extra cast. If you have typechecking that can understand that the cast in the init affects the methods, pls tell me how you got it.